app/src/main/java/org/p2p/wallet/transaction/model/progressstate/TransactionState.kt

```diff
@@ -0,0 +1,18 @@
+package org.p2p.wallet.transaction.model.progressstate
+
+import androidx.annotation.StringRes
+import org.p2p.wallet.R
+
+sealed class TransactionState {
+
+    open class Progress(
+        @StringRes val message: Int = R.string.send_transaction_being_processed,
+        @StringRes val description: Int = R.string.transaction_description_progress,
+    ) : TransactionState()
+
+    open class Success : TransactionState()
```

Comment:
can be object

Reply:
open is not applicable to object =( sorry